src/server/pfs/cmds/cmds.go

```diff
@@ -1201,18 +1203,32 @@ $ {{alias}} 'foo@master:/test\[\].txt'`,
 				if err != nil {
 					return err
 				}
-				f, err := progress.Create(outputPath, int64(fi.SizeBytes))
-				if err != nil {
-					return err
+				var f *progress.File
+				if ofi, err := os.Stat(outputPath); retry && err == nil {
```

Comment:
The right way to do this if not exists pattern is to use `os.O_EXCL` but it's not a big deal because Create can just error and the user will probably retry.
https://pkg.go.dev/os#pkg-constants

Reply:
I can restructure it but the reason I use os.Stat s to check file size if it exists